packages/vuetify/src/components/Vuetify/mixins/breakpoint.ts

```diff
@@ -1,121 +1,137 @@
 import Vue from 'vue'
-import { VuetifyBreakpoint } from 'vuetify/types'
+import { VuetifyUseOptions, VuetifyBreakpoint } from 'vuetify/types'
 
 /**
- * A modified version of https://gist.github.com/cb109/b074a65f7595cffc21cea59ce8d15f9b
+ * Factory function for the breakpoint mixin.
  */
+export default function breakpoint (opts: VuetifyUseOptions['breakpoint'] = {}) {
+  if (!opts) {
```

Comment:
Remove the default assignment? `VuetifyUseOptions['breakpoint']` includes `false | undefined`